web-common/src/features/dashboards/scheduled-reports/time-utils.ts

```diff
@@ -0,0 +1,72 @@
+export function getTodaysDayOfWeek(): string {
+  const dayOfWeekMap: Record<number, string> = {
+    0: "Sunday",
+    1: "Monday",
+    2: "Tuesday",
+    3: "Wednesday",
+    4: "Thursday",
+    5: "Friday",
+    6: "Saturday",
+  };
+
+  return dayOfWeekMap[new Date().getDay()];
```

Comment:
I would switch to using Luxon for most of these cases https://moment.github.io/luxon/api-docs/index.html#infoweekdays